### PR TITLE
EditorHelpBit: Fix not inheriting editor theme/fonts

### LINKED ISSUE #37676: Visual bugs with EditorHelpBit
**Godot version:**
b2f79cac9a4b69a24ccf5f2bfeaeb5b38b7db369

**Issue description:**
- Font size is incorrect.
- Text cuts off (likely due to the previous point).
- Bold and underline text is not set properly.

### PR DESCRIPTION
**Pull request by akien-mga**
Fixes https://github.com/godotengine/godot/pull/37164#issuecomment-635090418.
_Bugsquad edit:_ Fixes #37676.

Screenshot_20200528_162156

It seems to point at a bigger theme propagation issue though.
I'd expect that controls added as children to editor node should inherit the editor theme, but this doesn't seem to happen here. `EditorHelpBit` is a custom tooltip so I tried to make sure it reuses the theme of its parent control, but that didn't seem to work:
```diff
diff --git a/scene/main/viewport.cpp b/scene/main/viewport.cpp
index 23b559d3ec..ef66c11475 100644
--- a/scene/main/viewport.cpp
+++ b/scene/main/viewport.cpp
@@ -1539,6 +1539,8 @@ void Viewport::_gui_show_tooltip() {
        panel->set_transient(false);
        panel->set_flag(Window::FLAG_NO_FOCUS, true);
        panel->set_wrap_controls(true);
+       // Use same theme as parent control.
+       panel->set_theme(which->get_theme());
        panel->add_child(base_tooltip);
 
        gui.tooltip_popup = panel;
```

### CODE REVIEW

**Comment by txj-mssl** [comment]
Thanks, I tried using 9f5d4f5, EditorHelpBit: Fix not inheriting editor theme/fonts #39123, successfully making details The panel shows Chinese. If there are follow-up questions, I will continue to report.

**Comment by txj-mssl** [comment]
This afternoon, I downloaded the main branch, used this fix code, and tried it for a while. Basically, it can display normally, but there is a small problem, I don't know what's going on.
Project manager, when you are free, please see what's going on. I use piedot to edit Po files.
thank you.

**Comment by akien-mga** [comment]
That's not a bug, the text was changed recently (see how the one in editor uses `0.5`, while the msgid of the .po file is `50`).
As a matter of fact, I *just* updated the .pot and .po file 5 minutes ago, so once https://hosted.weblate.org/projects/godot-engine/godot-class-reference finishes syncing and is unlocked, you will be able to correct the translation and it will work fine.

**Comment by txj-mssl** [comment]
OK, I'll try to modify it after synchronization.

**Comment by akien-mga** [comment]
By the way, would you want to contribute your translations so that Chinese users can benefit from them when Godot 4.0 is released?

We handle class reference translations on Weblate at https://hosted.weblate.org/projects/godot-engine/godot-class-reference/zh_Hans/

Right now it's locked as it's syncing with the latest update: https://github.com/godotengine/godot/commit/ce5de2a3f49f152b799fe2a00cba4c6bd1b2e28c
It should be unlocked in ~30 min or 1 hour, depending on how long Weblate takes (it's slow to reset the repository for some reason).

Once it's unlocked, you could upload your `zh_CN.po` file on https://hosted.weblate.org/projects/godot-engine/godot-class-reference/zh_Hans/ using this option:
Screenshot_20200625_111443

Your file is based on the older `.pot` file, but I think Weblate will automatically merge your content with the new template. You can then re-download the merged `.po` file based on the current template in the same menu:
Screenshot_20200625_111746

(But again, this can't be done *right now* as I locked the repository. It will be unlocked soon.)

**Comment by akien-mga** [comment]
> What's your email address, I'd like to write to you.

https://github.com/godotengine/godot/issues/14582#issuecomment-633534379

**Comment by txj-mssl** [comment]
ariel@godotengine.org
Is this the address of the addressee?

**Comment by txj-mssl** [comment]
Or is it?
remi@godotengine.org

**Comment by akien-mga** [comment]
I'm Rémi yes, so it's `remi@godotengine.org` as written in https://github.com/godotengine/godot/issues/14582#issuecomment-633534379.

**Comment by txj-mssl** [comment]
ok.

**Comment by txj-mssl** [comment]
Although my English is not good, I am willing to contribute translation.
Because the business engine is always under control, no matter how friendly it is to developers. Recently, one thing has a great impact on me. The United States banned matlab from a university in China, which made me realize the possibility that learning commercial game engine may be affected by the ban in the United States, while open source will not be troubled by this aspect.

I'm a novice in game development. I've browsed hundreds of web pages, and I'm surprised to come to the conclusion that Godot has become the most popular open-source game engine at present. Other open-source game engines may be better in some aspects, but from the perspective of popularity and strength, Godot has become the game engine MIT No.1

In China, from my observation, there are often so many foreign game engines used. Unreal Engine 4, unity, gamemaker Studio 2, gamemaker MV. In the past, there was no Godot in this list, but now new developers in China prefer to use Godot.

The official says that Godot is small, but I think it will get bigger and bigger with the addition of new functions, so we may change the slogan at that time.

In summary, I would like to contribute to translation.
Sorry, I said a lot.
I hate people who say they want to ban me.

I hope the translation is in Chinese and English, the above is the original English, and the following is the translated Chinese.
this look
image
It's not like that
image

I found that the name of the Chinese Po file is “zh_Hans”，After the build runs, it doesn't work. I try to change it to zh_CN, then normal operation.

**Comment by akien-mga** [comment]
Superseded by #40107.
